testData/preferences/andrea.json5

```diff
@@ -0,0 +1,14 @@
+{

```

Comment:
A comment about this would be nice, also for norman.

Reply:
@JavierJF - bump

Reply:
This is addressed.